OpenRA.Mods.D2k/Traits/AttractsWorms.cs

```diff
@@ -0,0 +1,60 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor makes noise, which causes them to be targeted by actors with the Sandworm trait.")]
+	public class AttractsWormsInfo : ITraitInfo
+	{
+		[Desc("How much noise this actor produces.")]
+		public readonly int Intensity = 0;
+
+		[Desc("Noise percentage at Range step away from the actor.")]
+		public readonly int[] Falloff = { 100, 100, 25, 11, 6, 4, 3, 2, 1, 0 };
+
+		[Desc("Range between falloff steps.")]
+		public readonly WRange RangeStep = new WRange(4096);
```

Comment:
Set both to 27 cells so we don't get too greedy and choke on that `ActorsInRange`.